x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/open_close_signals_route.ts

```diff
@@ -66,7 +67,12 @@ export const setSignalsStatusRoute = (router: SecuritySolutionPluginRouter) => {
           refresh: true,
           body: {
             script: {
-              source: `ctx._source.signal.status = '${status}'`,
+              source: `if (ctx._source['${ALERT_WORKFLOW_STATUS}'] != null) {
+                ctx._source['${ALERT_WORKFLOW_STATUS}'] = '${status}'
+              }
+              if (ctx._source.signal != null && ctx._source.signal.status != null) {
+                ctx._source.signal.status = '${status}'
+              }`,
```

Comment:
I don't think we need this additional logic to check for workflow_status in this route since it's security solution specific, but the route will still function correctly with it here.